### DISCUSSION TITLE
What is the "upload status" notification for?

### DISCUSSION THREAD

**Discussion by 20GotoTen** [LawnchairLauncher/lawnchair#4380] [Q&A] [answered]
In the app info notification settings for Lawnchair, there's an entry for "upload status". Can someone shed light on what it's for please? Thanks.

**Comment by validcube** [comment, marked as answer]
Lawnchair will create a notification under the "Bug Reports" category to notify you when it's crashed with an option to upload the log to [katbin](https://katb.in/) (text storage site).

When you choose to upload the log, it will create another notification under a new category (in this case, "Upload Status").

https://github.com/LawnchairLauncher/lawnchair/blob/bffc1983df91e94328f7c3fd869e4d5685cbbc98/lawnchair/src/app/lawnchair/bugreport/UploaderService.kt#L59-L73
